src/app/workflow/info-tab/info-tab.component.ts

```diff
@@ -105,6 +106,7 @@ export class InfoTabComponent extends EntryTab implements OnInit, OnChanges {
   }
 
   ngOnInit() {
+    this.zenodoUrl = Dockstore.ZENODO_AUTH_URL ? Dockstore.ZENODO_AUTH_URL.replace('oauth/authorize', '') : '';
```

Comment:
`this.zenodoUrl = Dockstore.ZENODO_AUTH_URL?.replace('oauth/authorize', '') || '';` _might_ work if you want to be fancy.  same deal with the `selectedVersion` in the html

Reply:
The version of prettier/TS we're using doesn't like the `?.` syntax. Could only apply to the HTML.